### ISSUE TITLE
Error when running Load Testing -- in Locust

### ISSUE THREAD

**Issue by swcurran** [hyperledger/aries-mediator-service#67]
I'm getting this exception on the load-testing-agent. Any idea what it is?  Something with my setup?  I've got a pretty vanilla setup according to the instructions -- but maybe I didn't follow a step?  This is happening with the Mediator running on Docker, and Locust on Docker on the same machine.  I'm wondering if it is the `AGENT_IP`, as I didn't change it. Don't know what to change it to, or how to figure it out.

```
load-testing-load-agent-1  | [2023-04-03 18:35:36,265] load-agent/ERROR/locust.user.task: 'NoneType' object is not subscriptable
load-testing-load-agent-1  | Traceback (most recent call last):
load-testing-load-agent-1  |   File "/usr/local/lib/python3.8/dist-packages/locust/user/task.py", line 347, in run
load-testing-load-agent-1  |     self.execute_next_task()
load-testing-load-agent-1  |   File "/usr/local/lib/python3.8/dist-packages/locust/user/task.py", line 372, in execute_next_task
load-testing-load-agent-1  |     self.execute_task(self._task_queue.pop(0))
load-testing-load-agent-1  |   File "/usr/local/lib/python3.8/dist-packages/locust/user/task.py", line 384, in execute_task
load-testing-load-agent-1  |     task(self)
load-testing-load-agent-1  |   File "/load-agent/load-agent/locustIssue.py", line 33, in accept_invite
load-testing-load-agent-1  |     connection = self.client.accept_invite(self.invite['invitation_url'])
load-testing-load-agent-1  | TypeError: 'NoneType' object is not subscriptable
load-testing-load-agent-1  | 
load-testing-load-agent-1  | [2023-04-03 18:35:39,484] load-agent/ERROR/locust.user.task: 'NoneType' object is not subscriptable
load-testing-load-agent-1  | Traceback (most recent call last):
load-testing-load-agent-1  |   File "/usr/local/lib/python3.8/dist-packages/locust/user/task.py", line 347, in run
load-testing-load-agent-1  |     self.execute_next_task()
load-testing-load-agent-1  |   File "/usr/local/lib/python3.8/dist-packages/locust/user/task.py", line 372, in execute_next_task
load-testing-load-agent-1  |     self.execute_task(self._task_queue.pop(0))
load-testing-load-agent-1  |   File "/usr/local/lib/python3.8/dist-packages/locust/user/task.py", line 384, in execute_task
load-testing-load-agent-1  |     task(self)
load-testing-load-agent-1  |   File "/load-agent/load-agent/locustIssue.py", line 33, in accept_invite
load-testing-load-agent-1  |     connection = self.client.accept_invite(self.invite['invitation_url'])
load-testing-load-agent-1  | TypeError: 'NoneType' object is not subscriptable
load-testing-load-agent-1  | 
```

**Comment by swcurran** [comment]
@reflectivedevelopment -- any ideas?

@usingtechnology -- FYI.

**Comment by swcurran** [comment]
The key issue seems to be that the “sample.env” calls for the use of “issue” instead of “ping” as the Locust activity. Will update that as for Mediator testing, ping is what we want.

**Comment by swcurran** [comment]
Addressed in PR #74.

**Comment by reflectivedevelopment** [comment]
Merged #74
